### ISSUE TITLE
The `overridden_super_call` rule does not detect missing super call in `updateTrackingAreas()`

### ISSUE THREAD

**Issue by sindresorhus** [realm/SwiftLint#2616]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

The rule does not detect that the following is missing a `super` call:

```swift
class Button: NSButton {
	override func updateTrackingAreas() {
		// …
	}
}
```

It should be:

```swift
class Button: NSButton {
	override func updateTrackingAreas() {
		super.updateTrackingAreas()
	}
}
```

From Apple docs:

> You should override this method to remove out of date tracking areas and add recomputed tracking areas; **your implementation should call super.**
\- https://developer.apple.com/documentation/appkit/nsview/1483719-updatetrackingareas#

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint lint
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? 0.30.1
* Installation method used (Homebrew, CocoaPods, building from source, etc)? Installer from GitHub
* Paste your configuration file:

```yml
opt_in_rules:
  - overridden_super_call
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)? No
* Which Xcode version are you using (check `xcode-select -p`)? 10.1
* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
  to quickly test if your example is really demonstrating the issue. If your example is more
  complex, you can use `swiftlint lint --path [file here] --no-cache --enable-all-rules`.

```shell
echo "class Button: NSButton { override func updateTrackingAreas() {} }" | swiftlint lint --no-cache --use-stdin --enable-all-rules
```

**Comment by marcelofabri** [comment]
`overridden_super_call` manually checks against a set of methods.

You can add a custom method via the configuration:

```yml
overridden_super_call:
  included:
    - '*'
    - updateTrackingAreas()
```

**Comment by sindresorhus** [comment]
That is not documented anywhere from what I could find.

**Comment by sindresorhus** [comment]
What does `*` mean?

**Comment by sindresorhus** [comment]
I think `updateTrackingAreas()` should be included in the default list: https://github.com/realm/SwiftLint/blob/d1dbc31aa9269364d6d7f43d2f99c82e12ceca6f/Source/SwiftLintFramework/Rules/RuleConfigurations/OverridenSuperCallConfiguration.swift#L2

**Comment by marcelofabri** [comment]
@sindresorhus You're right, currently rule configurations are not very well documented - I had to look at the source code.

`*` means that the default list of methods checked will be kept. If you don't add it, the list will be overwritten by the configuration.

Here's the list of methods currently checked:

```swift
let defaultIncluded = [
        //NSObject
        "awakeFromNib()",
        "prepareForInterfaceBuilder()",
        //UICollectionViewLayout
        "invalidateLayout()",
        "invalidateLayout(with:)",
        "invalidateLayoutWithContext(_:)",
        //UIView
        "prepareForReuse()",
        "updateConstraints()",
        //UIViewController
        "addChildViewController(_:)",
        "decodeRestorableState(with:)",
        "decodeRestorableStateWithCoder(_:)",
        "didReceiveMemoryWarning()",
        "encodeRestorableState(with:)",
        "encodeRestorableStateWithCoder(_:)",
        "removeFromParentViewController()",
        "setEditing(_:animated:)",
        "transition(from:to:duration:options:animations:completion:)",
        "transitionCoordinator()",
        "transitionFromViewController(_:toViewController:duration:options:animations:completion:)",
        "viewDidAppear(_:)",
        "viewDidDisappear(_:)",
        "viewDidLoad()",
        "viewWillAppear(_:)",
        "viewWillDisappear(_:)",
        //XCTestCase
        "setUp()",
        "tearDown()"
    ]
```

**Comment by sindresorhus** [comment]
Please keep it open.

**Comment by sindresorhus** [comment]
Please keep it open.
